src/code.cloudfoundry.org/cf-tcp-router/routing_table/updater.go

```diff
@@ -244,19 +244,31 @@ func (u *updater) toRoutingTableEntry(logger lager.Logger, routeMapping apimodel
 		ttl = *routeMapping.TTL
 	}
 
+	var terminateFrontendTLS bool
+	if routeMapping.TerminateFrontendTLS != nil {
+		terminateFrontendTLS = *routeMapping.TerminateFrontendTLS
+	}
+
+	var alpn string
+	if routeMapping.ALPN != nil {
```

Comment:
routeMapping.ALPN is a string. It will never equal nil.

I don't see tests for this.

Reply:
I had defined this as a pointer variable in the gorm struct.

Are there any opinions as to when to use pointer variables?